CHANGELOG.md

```diff
@@ -10,3 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [Unreleased]: https://github.com/comit-network/comit-rs/compare/1625533e04119e8496b14d5e18786f150b4fce4d...HEAD
 
+- Update handling of config files
+
+[Changed]: For configuration files we now use the standard location as expected on each platform (OSX, Windows, Linux).
```

Comment:
This is not the expected format, see https://keepachangelog.com/en/1.0.0/

"Changed" should be a section. It should contain a list that are the changes.